### PR TITLE
feat: Add more aliases for GreaterThanFunction and GreaterThanOrEqualsFunction

### PR DESCRIPTION
**Pull request by kristofbolyai**

### CODE REVIEW

File: common/src/main/java/com/wynntils/functions/generic/LogicFunctions.java [hunk 1]
```diff
@@ -151,7 +151,7 @@ public FunctionArguments.RequiredArgumentBuilder getRequiredArgumentsBuilder() {
 
         @Override
         public List<String> getAliases() {
-            return List.of("mt");
+            return List.of("mt", "more_than", "gt");
```
Comment (magicus) [inline common/src/main/java/com/wynntils/functions/generic/LogicFunctions.java:154]:
```suggestion
            return List.of("mt", "more_than", "gt",">");
```

or do function names have to be a-z?

Comment (magicus) [reply common/src/main/java/com/wynntils/functions/generic/LogicFunctions.java:154]: Also, just checking, is less than already `lt`?

**Review by magicus — APPROVED** [review]

File: common/src/main/java/com/wynntils/functions/generic/LogicFunctions.java (hunk 1 above)
Comment (kristofbolyai) [reply common/src/main/java/com/wynntils/functions/generic/LogicFunctions.java:154]: Technically not, but this looks really bad imo.

Comment (kristofbolyai) [reply common/src/main/java/com/wynntils/functions/generic/LogicFunctions.java:154]: > Also, just checking, is less than already `lt`?

yes

**Pull request merged**